README.md

```diff
@@ -17,96 +12,42 @@ Clone the source code:
 git clone https://github.com/projecttacoma/measure-repository-service.git
 ```
 
-Install dependencies:
+Install root directory dependencies:
 
 ```bash
 npm install
 ```
 
-### MongoDB
-
-This test server makes use of [MongoDB](https://www.mongodb.com), a cross-platform document-oriented database program.
-
-Follow the [MongoDB Community Edition installation guide](https://docs.mongodb.com/manual/installation/) for your platform, and follow the commands for running MongoDB on your machine.
-
-### First time Database Setup
-
-This server comes with a script to create collections for FHIR Measure, Library, and MeasureReport resources which will be used by the measure repository service.
-To create these collections:
-
-```
-npm run db:setup
-```
-
-to clear the database and regenerate empty FHIR Measure, Library, and MeasureReport collections, run:
+### Frontend and Backend Installation
 
-```
-npm run db:reset
-```
+Refer to the READMEs for both the frontend and backend of the Measure Repository Service in their respective directories for specific setup and installation instructions.
```

Comment:
Thoughts on also adding a sentence in this README about how the repository uses `npm workspaces` and how all `npm` installations need to be done at the root directory?

Fine leaving this README as-is if you think this will make us sound like a broken record lol. But it seems like the flow will be: the user reads this README, which says to look at the backend/frontend READMEs, and then those READMEs say you need to be in the root directory for installations, and so the user then needs to `cd` back to the root dir. So it might be worth adding a blurb here as well to reduce the amount of `cd`-ing that needs to be done